index.html

```diff
@@ -12,26 +12,31 @@
 </head>
 
 <body>
-    <nav class="navbar navbar-static-top">
-        <a class="navbar-brand" href="index.html">OfekTwitter</a>
-        <ul class="nav navbar-nav navbar-right">
-            <li>
-                <a href="users.html">Users</a>
-            </li>
-            <li>
-                <a href="sign-up.html">Sign Up</a>
-            </li>
-            <li>
-                <a href="sign-in.html">Sign In</a>
-            </li>
-        </ul>
+    <nav class="navbar">
+        <div class="container-fluid">
+            <div class="navbar-header">
+                <a class="navbar-brand" href="#">OfekTwitter</a>
+            </div>
+
+            <ul class="nav navbar-nav navbar-right">
+                <li>
+                    <a href="users.html">Users</a>
+                </li>
+                <li>
+                    <a href="sign-up.html">Sign Up</a>
+                </li>
+                <li>
+                    <a href="sign-in.html">Sign In</a>
+                </li>
+            </ul>
+        </div>
     </nav>
 
-    <div class="mainPage">
-        <label>
+    <div class="container">
+        <div>
             <textarea name="tweetText" id="tweetText" cols="80" rows="3" placeholder="tweet text.."></textarea>
```

Comment:
instead of `<textarea>` you can use `<input>` + Bootstrap grid system + the "form-control" class

Reply:
It looks like the wanted results requires textarea (bootstrap-index.html image)